indexer/src/test/java/org/uniprot/store/indexer/crossref/CrossRefReaderTest.java

```diff
@@ -36,21 +35,36 @@ void testReadFile() {
         verifyDBXRef(dbxRef);
         assertEquals("Implicit", dbxRef.getLinkType());
         int count = 1;
-        while (reader.read() != null) {
+        String crossRefWithMultipleServers = "DB-0218";
+        boolean serversCrossRefVerified = false;
```

Comment:
This boolean variable is not doing anything, because if it fails inside verifyCrossRefWithMultipleServers method, the execution will never reach like 48.

Reply:
You're right about code not reaching in case of failur in verifyCrossRefWithMultipleServers. But the purpose of testing boolean serversCrossRefVerified as true on line 48 is to make sure that the code goes inside if block(line 42). We may not have id "DB-0218" in the input file and multiple servers might not have verified at all by verifyCrossRefWithMultipleServers.

Reply:
You could load it all into a list or map, assert the size 8, assert if the id exists in the list or map and validate it. It would be simpler.